frontend/cypress/tests/zoomView/zoomView_reload.js

```diff
@@ -9,10 +9,10 @@ describe('reload page', () => {
         .click();
 
     cy.get('#tab-zoom > .sorting > .sort-by > select:visible')
-        .select('Time');
+        .select('LoC');
```

Comment:
Why the default controls will be affected here? I didn't see the related code change?

Reply:
The default controls are not affected here. But the purpose of selecting LoC is to ensure that it remains selected upon reload.

Reply:
OK, understood